doc/content/getting-started/user-management/_index.md

```diff
@@ -1,6 +0,0 @@
----
-title: "Basic User Management"
```

Comment:
Deleting this breaks links in emails we're sending, so please keep it or alias it.